fixtures/browser.py

```diff
@@ -29,7 +29,7 @@ def pytest_runtest_setup(item):
 
 def pytest_exception_interact(node, call, report):
     name, location = get_test_idents(node)
-    val = unicode(call.excinfo.value)
+    val = unicode(call.excinfo.value.message.decode('utf-8'))
```

Comment:
and maybe keep the `"ignore"` param in it, just to play safe, away from the various UnicodeDecodeErrors and similar pleasant stuff.